src/quartic/pipeline/validator/dag_utils.py

```diff
@@ -0,0 +1,137 @@
+import importlib.util
+import pprint
+import os.path
+import networkx as nx
+from quartic.utils import QuarticException
+from ..step import Step
+from .utils import save_graphviz, save_json
+
+def build_dag(steps, default_namespace):
+    assert steps
```

Comment:
Agreed, but we can make this better later. This was just to quickly catch obviously wrong behaviour.